src/workloads/query/PercentilesAgg.yml

```diff
@@ -22,7 +22,7 @@ AutoRun:
       - standalone-sbe
     branch_name:
       $gte:
-      - v7.0
+        v7.0
```

Comment:
[nit] For clarity, I would consider putting the equality matches on one line ie. ```$gte: v7.0``` but functionally your version is the same.